.travis.yml

```diff
@@ -41,4 +41,4 @@ script:
   - RUBYOPT='-W:no-deprecated -W:no-experimental' bundle exec rspec
 
 after_script:
-  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+  - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
```

Comment:
Removing the line ending isn't a cleanup but linter will get it when we run that again after hacktober :)